acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/proxy/PassThroughAcceptanceTest.java

```diff
@@ -32,25 +32,34 @@
       Convert.toWei(TRANSFER_AMOUNT_ETHER, Convert.Unit.ETHER).toBigIntegerExact();
 
   @Test
-  public void ethGetTransactionCountReturnCorrectNumber() {
-    final BigInteger startTransactionCount =
+  public void ethGetTransactionCountReturnCorrectNumber() throws Exception {
```

Comment:
can remove the throws Exception on this test and the other one

Reply:
done